FortnitePorting/Plugins/Blender/import_task.py

```diff
@@ -563,14 +591,153 @@ def get_meta(search_props):
         match mesh_type:
             case "Body":
                 meta = get_meta(["SkinColor"])
-            case "Head":
-                meta = get_meta(["MorphNames", "HatType", "PoseData"])
+            case "Head" | "Face":
```

Comment:
We'll actually want to apply Pose Asset data to body (maybe just all armatures) as there is at least one case where the jaw clips into clothing otherwise (i.e. Cube Queen).

Reply:
agreed, we should probably add important pose data stuff to every meta type and carry out the pose asset stuff afterwards no matter the type

Reply:
Done, let me know what you think.